### ISSUE TITLE
Will the project be updated in the future?

### ISSUE THREAD

**Issue by meowlgm** [LSimon95/megatts2#17]
Dear author, are you still working on this project? Megatts2 is quite impressive, and I'm really grateful that you were able to implement it. I'm very much looking forward to seeing further progress on this project in the future.

**Comment by wwfcnu** [comment]
+1

**Comment by LSimon95** [comment]
Yes. But for now, I do not have enough GPU to train the new 24k version with a bigger dataset.
